docs/source/1.0/spec/core/type-refinement-traits.rst

```diff
@@ -11,60 +11,126 @@ the type of a shape.
     :backlinks: none
 
 
-.. smithy-trait:: smithy.api#box
-.. _box-trait:
+.. smithy-trait:: smithy.api#default
 
--------------
-``box`` trait
--------------
+.. _default-trait:
+
+-----------------
+``default`` trait
+-----------------
 
 Summary
-    Indicates that a shape is boxed. When a structure :ref:`member <member>` is
-    marked with this trait or the shape targeted by a structure member is marked
-    with the ``box`` trait, the member may or may not contain a value, and the
-    member has no :ref:`default value <default-values>`.
-
-    Boolean, byte, short, integer, long, float, and double shapes are only
-    considered boxed if they are marked with the ``box`` trait. All other
-    shapes are always considered boxed.
+    Marks a structure member as having a default, zero value.
 Trait selector
-    .. code-block:: none
-
-        :test(boolean, byte, short, integer, long, float, double,
-              member > :test(boolean, byte, short, integer, long, float, double))
+    ``structure > member :not(> :test(union, structure > :test([trait|required])))``
 
-    *A boolean, byte, short, integer, long, float, double shape or a member that targets one of these shapes*
+    A member of a structure that does not target a union or structure.
```

Comment:
The default trait works on structures so long as the structure has all optional members (no required members)